### PR TITLE
it's my first commit array babble sorting

### PR DESCRIPTION
**Pull request by vitaliiit**

### CODE REVIEW

File: src/ArrayE.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+public class ArrayE {
+
```
Comment (YuraLukianchenko) [inline src/ArrayE.java:2]: please reformat code

File: src/ArrayE.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+    public int[] sort(int[] array) {
+        for (int j = 0; j < array.length; j++) {
+            for (int i = 1; i < array.length; i++) {
```
Comment (YuraLukianchenko) [inline src/ArrayE.java:7]: please extract this code in separate method

File: src/ArrayE.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+                boolean leftGreaterThanRight = array[i - 1] > array[i];
+//            System.out.println( array[i - 1] + " > " + array[i] + ", it is " + leftGreaterThanRight);
+
+                if (leftGreaterThanRight == true) { // чому тут можна без '== true'
```
Comment (YuraLukianchenko) [inline src/ArrayE.java:13]: It will be better tu use smth like that:  if(leftGreaterThanRight)

File: src/ArrayE.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+    public int[] sort(int[] array) {
+        for (int j = 0; j < array.length; j++) {
+            for (int i = 1; i < array.length; i++) {
+
+
+                boolean leftGreaterThanRight = array[i - 1] > array[i];
+//            System.out.println( array[i - 1] + " > " + array[i] + ", it is " + leftGreaterThanRight);
```
Comment (YuraLukianchenko) [inline src/ArrayE.java:11]: please remove unnessesary comments

File: src/ArrayE.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+                if (leftGreaterThanRight == true) { // чому тут можна без '== true'
+
+                    int z;
```
Comment (YuraLukianchenko) [inline src/ArrayE.java:15]: you can assign value to z when initialize it.

File: src/Central.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        array[1] = 8;
+        array[2] = 7;
+        array[3] = 6;
+        array[4] = 5;
+        array[5] = 4;
+        array[6] = 3;
+        array[7] = 2;
+        array[8] = 1;
+        array[9] = 0;
+
```
Comment (YuraLukianchenko) [inline src/Central.java:16]: pls reformat this code

File: src/ArrayE.java (hunk 1 above)
Comment (vitaliiit) [reply src/ArrayE.java:2]: done!

File: src/ArrayE.java (hunk 3 above)
Comment (vitaliiit) [reply src/ArrayE.java:13]: I do this later

**Pull request merged**